packages/effection/src/mailbox.ts

```diff
@@ -0,0 +1,104 @@
+import { monitor, Operation } from 'effection';
+import { EventEmitter } from 'events';
+
+import { compile } from './pattern';
+export { any } from './pattern';
+
+type EventEmitterName = string | symbol;
```

Comment:
I think this symbol is already defined in the `events` module as `EventName`

Reply:
Yes, looks like!